### ISSUE TITLE
Registration

### ISSUE THREAD

**Issue by zrouse-Boundless** [MapStory/mapstory-meta#1228]
Helpful and Welcoming Workflow
- [x] Send welcome email after account is activated
- [x] fix links in welcome email. "Learn the Basics" should go to http://mapstory.org/getskills. "Community" should go to http://mapstory.org/getinvolved.
- [x] Image is broken
- [ ] Make "Build the Community" our matching orange
- [ ](not sure) - add overview video as embed in email: https://www.youtube.com/watch?v=Jis7zXhC4gU
  ![grabgrab](https://cloud.githubusercontent.com/assets/2573761/13437042/44e019cc-dfb0-11e5-8a21-d233b1ca0fa5.jpg)

**Comment by travislbrundage** [comment]
First one should be addressed by this PR: https://github.com/MapStory/mapstory-geonode/pull/138

Need to test if it's working with social and what's missing there, if anything.

**Comment by travislbrundage** [comment]
For real fixed this time with tests: https://github.com/MapStory/mapstory-geonode/pull/139

Still have to look into social, though.

**Comment by jonpmarino** [comment]
Facebook and Google - failing to get confirmation email. Post create signal on both use cases suggested as solution.

**Comment by Coop56** [comment]
These should be addressed by this PR - https://github.com/MapStory/mapstory-geonode/pull/206/files

**Comment by jonpmarino** [comment]
flagging for @davisc. Still sending email with old links

**Comment by jonpmarino** [comment]
reassigning to @davisc to work last mile token issue. @Coop56 has fixes images, links and text

**Comment by Coop56** [comment]
Reminder for myself - this one just needs tested and confirmed as working.

**Comment by Coop56** [comment]
I tested this again - links are still broken.  I'll look into why.

**Comment by Coop56** [comment]
This one should be good to go now, I changed it to {{ site.domain }} based on @davisc feedback but I don't think demo is updating.

**Comment by Coop56** [comment]
@jonpmarino @davisc I was looking into this one tonight.  I tried testing it a few times on demo but no emails seem to be sending at all from demo now.  No confirmation or welcome email goes out.  I tried looking for logs from our email server to see if there were any errors but I'm not sure which mail server we're using?  I looked around for a few of the common ones but I didn't see them.  Maybe we can look into this tomorrow @davisc?
